superset/utils/pandas_postprocessing/pivot.py

```diff
@@ -87,7 +87,7 @@ def pivot(  # pylint: disable=too-many-arguments,too-many-locals
     if not drop_missing_columns and columns:
         for row in df[columns].itertuples():
             for metric in aggfunc.keys():
-                series_set.add(str(tuple([metric]) + tuple(row[1:])))
+                series_set.add(tuple([metric]) + tuple(row[1:]))
```

Comment:
@zhaoyongjie do you have any reservations about this change? IMO we have pretty good test coverage, so if this change doesn't break existing tests I feel this change is good to go (if it does cause a regression it's just a sign that we needed more/better tests). So I feel like this change is warranted, as it simplifies the code.

Reply:
@villebro from the line of code seems to serialize the combination of tuples, this change(L104) has used vector difference on the columns so I think this PR is better than before performance, and Thanks @Usiel for this PR.